src/licensedcode/data/rules/lua.yml

```diff
@@ -0,0 +1,4 @@
+license_expression: mit-modern
+relevance: 22.97
```

Comment:
Same comment: please use a round number ... unless you can explain why 22.97 would be the best pick?